vpr/src/place/place.cpp

```diff
@@ -1385,7 +1390,7 @@ static void update_net_bb(const ClusterNetId net,
         }
     } else {
         //For large nets, update bounding box incrementally
-        int iblk_pin = cluster_ctx.clb_nlist.pin_physical_index(blk_pin);
+        int iblk_pin = pin_tile_index(blk_pin);
```

Comment:
Since this is a free function (which is good), we probably want to give it a somewhat more descriptive name. `pin_tile_index` could be miss-understood to mean the tile index (within the grid) which contains the specified pin.

I think it really means the pin index of the tile corresponding to the blk_pin. Perhaps rename it to something like `tile_pin_index(ClusterPinId pin)` where the fact that it takes a ClusterPinId in combination with its name make sit clear it returns the pin index with respect to the tile.

Reply:
Done